src/components/App.tsx

```diff
@@ -5449,6 +5467,7 @@ class App extends React.Component<AppProps, AppState> {
             ((probablySupportsClipboardBlob && elements.length > 0) ||
               (probablySupportsClipboardWriteText && elements.length > 0)) &&
               separator,
+            allowBinding && actionBindText,
```

Comment:
we shouldn't show this when clicking on canvas as we show element-specific actions only when clicking on elements

Reply:
But in this case it does make sense to show it when clicking on canvas too coz its related to both the elements and not just a single one ? similar to copy styles as png/svg. Also I am not sure why we don't show the actions when elements selected and clicked on canvas coz when user is selecting elements they want to use the actions on those elements so it makes more intutive to show those actions even when clicked on canvas

Reply:
> Also I am not sure why we don't show the actions when elements selected and clicked on canvas coz when user is selecting elements they want to use the actions on those elements so it makes more intutive to show those actions even when clicked on canvas

That cannot be the assumption. The standard is that clicking on canvas means you want canvas actions. Some apps even auto-deselect when click on canvas, and we _may_ wanna do so later as well.

Either way, the way Excalidraw currently works is that clicking on canvas _does not_ show element actions, no matter of selection, so we shouldn't break that assumption just for this action.

Reply:
(note that "element" in this context means the common bounding box)

Reply:
> (note that "element" in this context means the common bounding box)
No we show canvas actions for common bounding box as well thats why I said its confusing and we should element actions when selection when clicking within selection

Reply:
> No we show canvas actions for common bounding box as well thats why I said its confusing and we should element actions when selection when clicking within selection

Huh. Is that a recent regression or was I mistaken? Well, let's fix that in another PR.

Reply:
Not sure, will check, I just noticed it now and hence started the discussion